### PR TITLE
Change repo address in docs

### PR DESCRIPTION
**Pull request by dominicbetts**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ All the important developer concepts are maintained as we continue to improve Az
...
 
-Sign up [here](https://microsoft.qualtrics.com/jfe/form/SV_0V8Xr8y9c4mHUfb) to be kept up to date about the next generation of Azure IoT Edge features. In the meantime, be sure to check out the great features we just added to Azure IoT Edge, like [development packages](https://azure.microsoft.com/en-us/blog/azure-iot-gateway-sdk-packages/) and an [Azure Stream Analytics module](https://azure.microsoft.com/en-us/blog/announcing-azure-stream-analytics-on-edge-devices-preview/).
+Sign up [here](https://microsoft.qualtrics.com/jfe/form/SV_0V8Xr8y9c4mHUfb) to be kept up to date about the next generation of Azure IoT Edge features. In the meantime, be sure to check out the great features we just added to Azure IoT Edge, like [development packages](https://azure.microsoft.com/en-us/blog/iot-edge-packages/) and an [Azure Stream Analytics module](https://azure.microsoft.com/en-us/blog/announcing-azure-stream-analytics-on-edge-devices-preview/).
```
Comment (damonbarry) [inline README.md]: The link https://azure.microsoft.com/en-us/blog/iot-edge-packages/ currently returns a 404 error. When will it be available?

File: README.md [hunk 2]
```diff
@@ -96,7 +96,7 @@ instructions used to create Azure IoT Edge Snap package. The walk-through is a g
 starting point for anyone trying to integrate Azure IoT Edge into their own snap
 package.
 
-API documentation can be found [here](http://azure.github.io/azure-iot-gateway-sdk/api_reference/c/html).
+API documentation can be found [here](http://azure.github.io/iot-edge/api_reference/c/html).
```
Comment (damonbarry) [inline README.md]: This API doc link points specifically to the stuff for C. Can we link instead to the higher-level page (http://azure.github.io/iot-edge/)?

File: samples/simulated_device_cloud_upload/README.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ The logger module logs all the exchanges on the message broker to a file.
 ### Linux
 
 1. Open a shell
-2. Navigate to `azure-iot-gateway-sdk\tools\`
+2. Navigate to `iot-edge\tools\`
```
Comment (damonbarry) [inline samples/simulated_device_cloud_upload/README.md]: Forward slashes for Linux

File: samples/simulated_device_cloud_upload/README.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ The logger module logs all the exchanges on the message broker to a file.
 ### Windows
 
 1. Open a Developer Command for VS2015 as an Administrator
-2. Navigate to `azure-iot-gateway-sdk/tools/`
+2. Navigate to `iot-edge/tools/`
```
Comment (damonbarry) [inline samples/simulated_device_cloud_upload/README.md]: Backslashes for Windows

File: README.md (hunk 1 above)
Comment (damonbarry) [inline README.md]: Actually I just merged #270, which updates this link. So this particular chunk can be reverted.

**Comment by dominicbetts** [comment]
@damonbarry - I made some changes to address your comments - thanks.

**Pull request merged**